sdk/storage/azure-storage-stress/src/main/java/com/azure/storage/stress/ContentInfo.java

```diff
@@ -0,0 +1,54 @@
+package com.azure.storage.stress;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.zip.CRC32;
+
+public class ContentInfo {
+    private final long length;
+    private final long crc;
+    private final ByteBuffer head;
+
+    public ContentInfo(long crc, long length, ByteBuffer head) {
```

Comment:
Does anything call this other than the factory method? If not, let's make this package private.